test/e2e/storage/vsphere/k8s_utils.go

```diff
@@ -0,0 +1,240 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+type KubeletOpt string
+
+const (
+	NodeStateTimeout            = 1 * time.Minute
+	KStart           KubeletOpt = "start"
+	KStop            KubeletOpt = "stop"
+	KRestart         KubeletOpt = "restart"
+)
+
+// podExec wraps RunKubectl to execute a bash cmd in target pod
+func podExec(pod *v1.Pod, bashExec string) (string, error) {
+	return framework.RunKubectl("exec", fmt.Sprintf("--namespace=%s", pod.Namespace), pod.Name, "--", "/bin/sh", "-c", bashExec)
+}
+
+// kubeletCommand performs `start`, `restart`, or `stop` on the kubelet running on the node of the target pod and waits
+// for the desired statues..
+// - First issues the command via `systemctl`
+// - If `systemctl` returns stderr "command not found, issues the command via `service`
+// - If `service` also returns stderr "command not found", the test is aborted.
+// Allowed kubeletOps are `KStart`, `KStop`, and `KRestart`
+func kubeletCommand(kOp KubeletOpt, c clientset.Interface, pod *v1.Pod) {
```

Comment:
This func appears to be copied from _e2e/storage/persistent-volumes_disturptive.go_. Please call the original func or if that's not possible move the original to a more accessible location. Almost always there are  "out-of-sync" problems when code is replicated.

Reply:
If vsphere test needs to call function in e2e/storage then it is creating cyclic imports.
Is it fine if I move this function into e2e/storage/utils/utils.go and use it?